source/overview/product.rst

```diff
@@ -24,31 +24,30 @@ Mattermost Editions
 
 Mattermost is offered in three editions:
 
-- **Mattermost Team Edition** - An open source "virtual office" for your team under MIT license, where you get things done day-to-day. Includes open source native mobile and desktop apps, continuous archiving, unlimited search, threaded messaging, custom branding, custom emoji and full access to webhooks, APIs, drivers and third party integrations. Get started by `downloading the open source server <https://about.mattermost.com/download>`__
+Mattermost Team Edition
+^^^^^^^^^^^^^^^^^^^^^^^
 
-- **Mattermost Enterprise Edition E10** - A commercial "virtual campus" for your teams, groups and departments working on multiple projects, scaling to thousands of users. This is a commercial extension to the open source Team Edition with commercial support, corporate directory integration via AD/LDAP, advanced access controls, and security. Get started with a `free trial license <https://about.mattermost.com/trial/>`__.
+A free-to-use, open source, self-hosted alternative to proprietary SaaS messaging. Team Edition is your open source "virtual office", offering all the core productivity benefits of competing SaaS solutions. It deploys as single Linux binary with MySQL or PostgreSQL under an MIT license. Features include:
 
-- **Mattermost Enterprise Edition E20** - A "virtual city" for your whole enterprise, scaling to tens of thousands of users. This commercial version includes all the features of Team Edition and E10 and adds advanced compliance, monitoring and scalability support, plus additional support benefits, including the option to purchase Premier Support with our highest-level support service level agreements. Get started with a `free trial license <https://about.mattermost.com/trial/>`__.
-
-Details of each offering are as follows:
-
-**Mattermost Team Edition** - A free-to-use, open source, self-hosted alternative to proprietary SaaS messaging. Team Edition is your open source "virtual office", offering all the core productivity benefits of competing SaaS solutions. It deploys as single Linux binary with MySQL or PostgreSQL under an MIT license. Features include:
-
-- One-to-one and group messaging, file sharing and unlimited search history
+- One-to-one and group messaging, file sharing, and unlimited search history
 - Native apps for iOS, Android, Windows, Mac, Linux
-- Threaded messaging, emoji and custom emoji
+- Threaded messaging, emoji, and custom emoji
 - Tools for custom branding
-- Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`__ and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`__
+- Continuous archiving and unlimited search
```

Comment:
`unlimited search history` is mentioned in the first bullet - not sure if this is a duplicate or if you have another use case in mind?

Reply:
No; oversight on my part. Thanks for picking that up!